x-pack/plugins/rollup/public/crud_app/_crud_app.scss

```diff
@@ -5,10 +5,3 @@
   align-items: flex-end; /* 1 */
 }
 
-/**
- * 1. Ensure panel fills width of parent when search input yields no matching rollup jobs.
- */
-.rollupJobsListPanel {
```

Comment:
I don't think this is necessary anymore; I included a screenshot in the PR description when there are no matching results.